pkg/security/probe/activity_dump_manager.go

```diff
@@ -24,9 +24,6 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/security/probe/dump"
 	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
 	"github.com/DataDog/datadog-agent/pkg/security/utils"
-
-	// util.GetHostname(...) will panic without this import
```

Comment:
Are we sure this provider is correctly initialized now ? What is the reasoning behind removing this code ?

Reply:
The old provider do not exist anymore, so no need to initialize anything.
The `hostname` resolution code does not rely on the new provider either.

Reply:
Oh great then !

Reply:
I added our team label on this PR so that we can QA it carefully since it can have some impact